### PR TITLE
Check for filetype while dragging.

### LINKED ISSUE #36: Herausziehen von Files (abspeichern)
Beim herausziehen von Files (abspeichern) wird der Uploader aktiviert. Der Uploader sollte nur aktiv sein, wenn ein File reingezogen wird.

### PR DESCRIPTION
**Pull request by bierik**
Check for file type while dragging.
Fixes #36

### CODE REVIEW

File: ftw/file/resources/uploader.js [hunk 1]
```diff
@@ -58,9 +58,11 @@ var uploader = (function($) {
       unbindEvents();
       if (tests.dnd) {
         $(document).on('dragenter', function(event) {
-          dragging++;
-          overlay.overlay().load();
-          event.preventDefault();
+          if(event.dataTransfer.types[0] === 'Files') {
```
Comment (maethu) [inline ftw/file/resources/uploader.js]: @bierik Does this also work for Images?

**Comment by maethu** [comment]
@bierik Please add changlog entry.

File: ftw/file/resources/uploader.js (hunk 1 above)
Comment (bierik) [inline ftw/file/resources/uploader.js]: I just tested with images and it worked fine. According to this doc it works for all file types. https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer#types.28.29

**Comment by jone** [comment]
@bierik please update :wink:

**Comment by jone** [comment]
This prevents displaying the uploader when dragging images of the site.
The uploader should only trigger when a file from the file system is dragged into the page.

**Pull request merged**